docker/fedora.Dockerfile

```
@@ -1,9 +1,7 @@
 FROM fedora:latest
 
-# Update system and install Flatpak
-RUN dnf -y update && \
+RUN dnf update -y && \
     dnf install -y flatpak flatpak-builder xorg-x11-server-Xvfb python3-ruamel-yaml && \
```

```suggestion
    dnf install -y flatpak flatpak-builder xorg-x11-server-Xvfb  && \
```
This one isn't needed anymore

---

Okay